.github/workflows/build.yaml

```
@@ -27,6 +27,8 @@ on:
       - "hacs.json"
       - ".github/workflows/build.yaml"
       - "CHANGELOG.md"
+    paths-ignore:
```

actionlint (1.7.12)

[error] both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)